DemoServer/AdditionalLanguages/python/multiMapIndexes/multiMapIndexCustomizedFields/metadata.json

```diff
@@ -0,0 +1,203 @@
+                },
+                {
+                    "Title": "Multi Map indexes from the client",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#multimap-indexes-from-the-client"
+                }
+            ]
+        },
+        {
+            "Title": "The index constructor",
+            "Slug": "step-4",
+            "DescriptionLines": [
+                "* Use `add_map()` to define the content of the index-fields per collection indexed.<br>",
+                "  Data from the different collections is indexed into the _common index-fields_ in the IndexEntry class.",
+                "",
+                "* The data is customized as needed when configuring each index field, e.g:",
+                "  * Field _contact_name_ is the combination of 2 raw document properties in the Employees collection.<br>",
+                "  * Field _collection_ is calculated from the document metadata.",
+                "  * The document properties are case-sensitive, use the exact casing as in the document.",
+                "",
+                "* The index will contain a mapping from any indexed term to the relevant document in either collection indexed.",
+                "",
+                "* When querying this index, the indexed terms can be used to filter the query results and get matching documents from either collection."
+            ],
+            "Assets": [
+                {
+                    "Title": "Querying many sources",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#querying-many-sources-at-once-with-multimap-indexes"
+                },
+                {
+                    "Title": "Multi Map indexes from the client",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#multimap-indexes-from-the-client"
+                }
+            ]
+        },
+        {
+            "Title": "Store index fields",
+            "Slug" : "step-5",
+            "DescriptionLines": [
+                "* Usually, when querying an index, the index provides the matching document ID, and results are taken from that document in the documents store.",
+                "",
+                "* _If the index-fields are customized_, and you want the query to return their content,<br>",
+                "  then these fields must be stored in the index.<br>",
+                "  The results will be taken directly from the index fields instead of from the document.",
+                "",
+                "* Call `store()` with param _FieldStorage.YES_ to store the data on the index."
```

Comment:
https://github.com/ravendb/demo/pull/193/files#r1350181511
it's a good line of documentation, that we might lack in the place I've annotated before